akka-cluster/src/main/scala/akka/cluster/ClusterRemoteWatcher.scala

```diff
@@ -70,6 +70,9 @@ private[cluster] class ClusterRemoteWatcher(
 
   override val log = Logging(context.system, ActorWithLogClass(this, ClusterLogClass.ClusterCore))
 
+  // allowed to watch even though address not in cluster membership, i.e. remote watch
+  private val watchPathWhitelist = Set("/system/sharding/")
```

Comment:
Basically a constant so could be in companion instead.

Reply:
true, but there is only one instance of `ClusterRemoteWatcher` so shouldn't matter